corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -174,7 +191,7 @@
             {% endif %}
             {% if user.can_edit_data %}
             <p>
-                {% if not is_archived %}
+                {% if not is_archived and not edit_info.was_edited %}
```

Comment:
This looks like it prevents a user from archiving an edited form, is there a reason for that?